.env.local.example

```diff
@@ -4,3 +4,7 @@ SECRET_KEY=
 # ADMIN CREDENTIALS
 ADMIN_USERNAME=
 ADMIN_PASSWORD=
+
+CLOUDINARY_CLOUD_NAME=your-cloud-name
+CLOUDINARY_API_KEY=your-api-key
+CLOUDINARY_API_SECRET=your-api-secret
```

Comment:
can you remove right side values from this `.example` file?
also add same keys in `.env.example` as well.
and maybe a comment above?
```bash
# CLOUDINARY
...
```